<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QUADrallegiance</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="widget_battery" native="true">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>90</y>
      <width>190</width>
      <height>141</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_battery">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>170</width>
       <height>111</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="progressBar_battery">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>54</y>
        <width>150</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>70</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_timeText">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>84</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>14</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Time:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_time">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>83</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>16</pointsize>
       </font>
      </property>
      <property name="text">
       <string>06:38</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_batteryTotalLabel">
      <property name="geometry">
       <rect>
        <x>33</x>
        <y>4</y>
        <width>30</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Total</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_batteryLowCellLabel">
      <property name="geometry">
       <rect>
        <x>104</x>
        <y>4</y>
        <width>50</width>
        <height>15</height>
       </rect>
      </property>
      <property name="text">
       <string>Low Cell</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_batteryTotal">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>20</y>
        <width>68</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>18.88</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_batteryLowCell">
      <property name="geometry">
       <rect>
        <x>102</x>
        <y>20</y>
        <width>54</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>21</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>8.88</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_battery">
     <property name="geometry">
      <rect>
       <x>68</x>
       <y>5</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="text">
      <string>BATTERY</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_throttle" native="true">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>70</width>
      <height>221</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_throttle">
     <property name="geometry">
      <rect>
       <x>2</x>
       <y>2</y>
       <width>66</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>THROTTLE</string>
     </property>
    </widget>
    <widget class="QFrame" name="frame_throttle">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>22</y>
       <width>46</width>
       <height>190</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QProgressBar" name="progressBar_throttle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>26</width>
        <height>150</height>
       </rect>
      </property>
      <property name="value">
       <number>56</number>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="textVisible">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_throttleValue">
      <property name="geometry">
       <rect>
        <x>4</x>
        <y>6</y>
        <width>38</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>54%</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_measurements" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>181</width>
      <height>81</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_measurements">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>22</y>
       <width>171</width>
       <height>58</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_height">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>10</y>
        <width>65</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>19.6'</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_heightText">
      <property name="geometry">
       <rect>
        <x>105</x>
        <y>38</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_distance">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>10</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>117'</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_distanceText">
      <property name="geometry">
       <rect>
        <x>14</x>
        <y>38</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Distance</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_measurements">
     <property name="geometry">
      <rect>
       <x>17</x>
       <y>2</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>GND. MEASUREMENTS</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_coordinates" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>160</width>
      <height>101</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_coordinates">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>160</width>
       <height>81</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="label_longitude">
      <property name="geometry">
       <rect>
        <x>15</x>
        <y>7</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>N 43Â° 2' 29.71&quot;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_latitude">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>W 87Â° 54' 9.993&quot;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_speed">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>52</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>0.72 kn @ 161.2Â°</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_coordinates">
     <property name="geometry">
      <rect>
       <x>35</x>
       <y>2</y>
       <width>90</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>COORDINATES</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_direction" native="true">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>161</width>
      <height>110</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_direction">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>161</width>
       <height>90</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QGraphicsView" name="graphicsView_direction">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>1</y>
        <width>159</width>
        <height>88</height>
       </rect>
      </property>
      <property name="interactive">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_direction">
     <property name="geometry">
      <rect>
       <x>47</x>
       <y>2</y>
       <width>67</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>DIRECTION</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_misc" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>51</width>
      <height>211</height>
     </rect>
    </property>
    <widget class="QFrame" name="frame_misc">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>22</y>
       <width>51</width>
       <height>189</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QRadioButton" name="radioButton_led1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_led1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>5</y>
        <width>38</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LED 1</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_led2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>16</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checkable">
       <bool>true</bool>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
      <property name="autoExclusive">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_led2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>45</y>
        <width>38</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>LED 2</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_srv2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>160</y>
        <width>40</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>93</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_srv2">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>140</y>
        <width>38</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SRV 2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_srv1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>85</y>
        <width>38</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>SRV 1</string>
      </property>
     </widget>
     <widget class="QProgressBar" name="progressBar_srv1">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>105</y>
        <width>40</width>
        <height>23</height>
       </rect>
      </property>
      <property name="value">
       <number>24</number>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_misc">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>2</y>
       <width>32</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>MISC.</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
